Can't build from GitHub source archives

## Expected Behavior

Allow building the node from github source archive

## Actual Behavior

Errors during build

## Steps to Reproduce the Problem

1. Download node source from github release
2. `make prod-package`

## Logs, error output, etc.

```
Epoch source archives (zip & tar; v1.0.x.*) in GitHub cannot be compiled, because of GitHub hooks issues around `base58` and `enacl`:

```
===> Compiling enacl
make[1]: Entering directory '/home/master/epoch_tar_deb/epoch-1.0.0/_build/default/lib/enacl/c_src'
make[1]: '/home/master/epoch_tar_deb/epoch-1.0.0/_build/default/lib/enacl/c_src/../priv/enacl_nif.so' is up to date.
make[1]: Leaving directory '/home/master/epoch_tar_deb/epoch-1.0.0/_build/default/lib/enacl/c_src'
===> Compiling enoise
===> Compiling ranch
===> Compiling cowboy
===> Compiling base58
fatal: Not a git repository (or any of the parent directories): .git
===> Hook for compile failed!

Makefile:320: recipe for target 'internal-build' failed
```

This step passes when the source is cloned from git tagged release.
```


## Specifications

 Unknown

--- a/Makefile
+++ b/Makefile
@@ -230,7 +230,7 @@ $(CT_DB_TARGETS):
 	$(MAKE) internal-ct
 
 REVISION:
-	@git rev-parse HEAD > $@
+	@git rev-parse HEAD > $@ || echo "unknown" > $@
 
 eunit-%: KIND=test
 eunit-%: internal-build

